### DISCUSSION TITLE
OCR功能失效  已经恢复  感谢作者!!

### DISCUSSION THREAD

**Discussion by xie0027** [shuax/MouseInc.Settings#329] [Q&A]
近几天开始OCR功能一直提示未知错误   作者有空可以修复一下吗？  万分感谢

**Comment by rolandfang** [comment]
遇到同样的问题，请作者修复一下，感谢

**Reply by xiaochenlius** [reply]
未知错误+1

**Reply by iDreamer11** [reply]
之前还能用的，今天手欠更新了一下系统，发现就不能用了

**Reply by 6Tom** [reply]
跟系统更新没有任何关系，微软成天背黑锅

**Reply by DaKaXiong** [reply]
![image](https://user-images.githubusercontent.com/105265244/187073349-c738f50a-7047-456e-bdbd-1bf2d5e98959.png)
一样的，也是最近几天遇到的问题，不知道是什么原因

**Reply by myxiaoshen** [reply]
ocr功能失效咯，能修复一下吗？如服务器问题我们能自定义配置api接口也行。🙇‍这个功能感觉非常实用，已经成了依赖了！突然识别失败有点反应不过来了。望开发者大大看到！提示：发生未知错误，请稍后再试。

**Comment by gbsix** [comment]
发生未知错误，请稍后在试

**Comment by a137933** [comment]
一样，不知道大家还有没有类似的ocr软件

**Comment by 6Tom** [comment]
ocr调用的是有道的接口，估计是演示接口达到数量上限，或者改参数，或者改规则了吧，说不准以后都没得白嫖了

**Reply by wistone777** [reply]
如果能使用qq或者微信客户端的截图识别就好了

**Reply by 6Tom** [reply]
可别，我宁可用panda、天若，也不要这一堆流氓软件

大家别急，可以先用替代品暂且代替一段时间，微软自家的开源工具powertoys正在开发ocr，我国提供了大规模客户，应该会考虑中文识别的，到那时候就没别人什么事了

**Reply by andydznj** [reply]
怎么恢复的？不好使啊

**Reply by uiop321** [reply]
ocr 用不了， 不稳定。显示错误，

**Comment by Eunknight** [comment]
https://github.com/shuax/MouseInc.Settings/discussions/335#discussioncomment-3523123
